cli/cmd/release_lint.go

```diff
@@ -0,0 +1,48 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/replicatedhq/replicated/cli/print"
+	"github.com/spf13/cobra"
+)
+
+var lintReleaseYaml string
+
+var releaseLintCmd = &cobra.Command{
+	Use:   "lint",
+	Short: "Lint a YAML",
+	Long:  "Lint a YAML",
+}
+
+func init() {
+	releaseCmd.AddCommand(releaseLintCmd)
+
+	releaseLintCmd.Flags().StringVar(&lintReleaseYaml, "yaml", "", "The YAML config to lint. Use '-' to read from stdin")
+}
+
+func (r *runners) releaseLint(cmd *cobra.Command, args []string) error {
+	if lintReleaseYaml == "" {
+		return fmt.Errorf("yaml is required")
+	}
+
+	if lintReleaseYaml == "-" {
```

Comment:
it looks like stdin is required? do you want to make it support a file path or perhaps ditch the flag? or am i missing something?

Reply:
This is the same behavior as "CreateRelease". I would like to support a file path! But thoughts on keeping this consistent for now, and handling that separately?

Reply:
sure. the flag just seems weird but im all for consistency